### ISSUE TITLE
Upload images and overlay on top of pdf files to be able to have drag and drop interactions.

### ISSUE THREAD

**Issue by jorgelondo** [bigbluebutton/bigbluebutton#17503]
Upload images and overlay on pdf files, to be able to have drag and drop interactions.
on some platforms you can copy the image from your computer and paste it with CTRL+V directly on the platform

**Comment by ffdixon** [comment]
We want this too.

Given that tldraw.com provides pasting of images, and we're building on top of tldraw, you can expect that we'll support pasting of images at some point.  Right now, the challenge is storage of the images on BigBlueButton server when pasting into BigBlueButton.

**Comment by Piotr-Fr** [comment]
Yes, very much, please.

**Comment by jorgelondo** [comment]
What good news!
if there was a "media library" it would be possible to upload it in that folder.

Thank you very much for this excellent platform.

**Comment by ffdixon** [comment]
The ability to paste in images is inherent in tldraw (the above screenshot was taken form tldraw.com), but we don't support pasting of images in BigBlueButton yet.  We need to add the server-side support to accept the uploaded image and sync it with other users, and support it in the recordings.

All work planned for the future.

**Comment by jorgelondo** [comment]
I understand @ffdixon , thank you very much!

**Comment by jorgelondo** [comment]
![Screenshot_2023-07-26-01-13-47-993-edit_com android chrome](https://github.com/bigbluebutton/bigbluebutton/assets/672505/f94add36-5cb0-4a37-931b-fd200ef4a043)
implement IA to generate images that can be used on top of the presentation.  In a previous publication, I had suggested being able to upload images that can be dragged, rotated, like any graphic element.  This to make the classes more interactive, in addition to uploading images, it would be ideal to be able to generate them with AI if the teacher or students require an image.

**Comment by jorgelondo** [comment]
The ability to paste images into BigBlueButton would make a big difference as it would allow drag-and-drop interactions over of presentations, is that in the roadmap?
Have they made any progress in this regard?
Thank you very much, the platform is spectacular!

**Comment by ffdixon** [comment]
> The ability to paste images into BigBlueButton would make a big difference as it would allow drag-and-drop interactions over of presentations, is that in the roadmap?

Yes, starting with the ability to paste images into the whiteboard.  We've not started the work yet, but it's on the road map.
